test/WormholeReceiver.t.sol

```diff
@@ -17,22 +17,71 @@ contract WormholeReceiverTestHarness is WormholeReceiver {
     bytes32 deliveryHash
   ) public virtual override {}
 
-  function modifierTest() public onlyRelayer {}
+  function onlyRelayerModifierFunc() public onlyRelayer {}
```

Comment:
We should follow the `exposed_` naming convention from the foundry best practices whenever a harness has a function that just exposes some internal function or modifer. There may be other places to change this also. Can be done in a separate PR

Reply:
https://github.com/ScopeLift/l2-flexible-voting/issues/45